kqueen/engines/jenkins.py

```diff
@@ -118,6 +151,17 @@ def provision(self, **kwargs):
         # PATCH THE CTX TO CONTAIN CLUSTER NAME AND UUID
         ctx[cluster_name] = 'kqueen-{}'.format(self.cluster.id)
         ctx[cluster_uuid] = self.cluster.id
+        if self.job_override_params:
```

Comment:
I suggest to manage all job parameters in a one place, so first let's add cluster_name and cluster_uuid to self.job_override_params (also for me self.job_parameters sounds better) and then manage everything in the same way

Reply:
imo its different cases - override-parameters used for overriding sane-default parameters, like boolean variables, checkers, additional cases
cluster name and UUID - must have cases, and it doesnot have any sane-defaults, so i prefer to manage it from separate UI-fields